components/Button/button.html

```diff
@@ -15,7 +15,7 @@
     <button class="btn primary">Primary</button>
     <button class="btn secondary">Secondary</button>
     <button class="btn error">Error</button>
-    <button class="btn success">Success</button>
+    <button class="btn success disabled" disabled>Success</button>
```

Comment:
Just a small suggestion, you can use the `:disabled` attribute instead of a class here for styling in CSS.

Reply:
Sure, on it!